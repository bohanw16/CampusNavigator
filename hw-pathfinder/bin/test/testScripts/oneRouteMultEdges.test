# find the shortest path with only one route

CreateGraph g

AddNode g n1
AddNode g n2
AddNode g n3

AddEdge g n1 n2 1.111
AddEdge g n1 n2 2.222
AddEdge g n2 n3 3.333
AddEdge g n2 n3 4.444

FindPath g n1 n3