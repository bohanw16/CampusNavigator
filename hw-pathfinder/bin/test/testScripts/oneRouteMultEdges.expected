# find the shortest path with only one route

created graph g

added node n1 to g
added node n2 to g
added node n3 to g

added edge 1.111 from n1 to n2 in g
added edge 2.222 from n1 to n2 in g
added edge 3.333 from n2 to n3 in g
added edge 4.444 from n2 to n3 in g

path from n1 to n3:
n1 to n2 with weight 1.111
n2 to n3 with weight 3.333
total cost: 4.444